MapboxCoreNavigation/NavigationService.swift

```diff
@@ -418,6 +418,16 @@ extension MapboxNavigationService: CLLocationManagerDelegate {
         //Finally, pass the update onto the router.
         router.locationManager?(manager, didUpdateLocations: locations)
     }
+    
+    public func locationManagerDidChangeAuthorization(_ manager: CLLocationManager) {
+        if #available(iOS 14.0, *) {
+            NotificationCenter.default.post(name: .locationAuthorizationDidChange, object: nil)
```

Comment:
I didn't send any UserInfo, but I can add that if it's necessary for consistency. It seems like a better idea to check for the authorization higher in the stack if a developer wants to observe this notification.

Reply:
It would be helpful to include user info so that developers don’t need to look around for the location manager to get the authorization status. I’m a bit concerned that the developer’s first reaction would be to create a throwaway location manager, which would be wasteful. If we don’t include any user info, then there’s no reason to define `locationAuthorizationKey`.

Reply:
Similar to other code in this PR, we can set `locationAuthorizationKey` to `manager.value(forKey: "accuracyAuthorization")`, without checking the value in any way.